### ISSUE TITLE
Riftbreaker editor - unable to create new research category

### ISSUE THREAD

**Issue by SenorRagequit** [exorstudios/riftbreaker-tools#93]
**Describe the bug**
Maybe I don't see it, but I seem to be unable to create a new research category in the Riftbreaker editor.
The gui itself is there, but just a filter box, no box where you can enter a name and click "Create".

**To Reproduce**
1. Open research editor of the Riftbreaker editor
2. Add new category for research

**Expected behavior**
A box to enter a name

**Screenshots**
![grafik](https://user-images.githubusercontent.com/50502395/156857364-b7a18bfb-1bd6-433c-81a1-5e8f83c629bb.png)

**Desktop (please complete the following information):**
Steam
Tools v. 4950

**Comment by lukaasm** [comment]
yes, ResearchEditor is still localization driven and shows lists based on existing localization keys.

We need add helper layer for creating new localization id's for categories/node its etc from within editor
